elastic_search/main.tf

```diff
@@ -145,73 +113,60 @@ resource "aws_cloudwatch_log_resource_policy" "example" {
 CONFIG
```

Comment:
I'm fine with leaving this like this, but in other modules we have set policies like this in a separate json file. Might be worth doing that just for the sake of consistency.

Reply:
This is very small. If it is big, I agree with what you mention.